emva1288/unittests/test_results.py

```diff
@@ -300,3 +302,68 @@ def test_nans(self):
         self.assertIs(r.DSNU1288_DN(), np.nan)
 
         del r
+
+
+class TestRoutines(unittest.TestCase):
+
+    # instantiate a bayer filter and a wrong filter
+    bayer_filter = get_bayer_filter(0, 1, 1, 0, 1000, 1000)
+    wrong_filter = np.tile([[1, 1, 1, 0], [1, 0, 0, 1]], (500, 250))
+
+    # intantiate masked images based in the masks
+    img = np.abs(np.random.random([1000, 1000]))
+    wrong = np.ma.array(img, mask=wrong_filter)
+    masked = np.ma.array(img, mask=bayer_filter)
+
+    # save the filtered images, masked and with no mask
+    res = high_pass_filter(img, 5)
+    img_filt = res['img']
+    im_num = res['multiplicator']
+    res = high_pass_filter(masked, 5)
+    ma_filt = res['img']
+    ma_num = res['multiplicator']
+
+    # introduce a low frequency signal in the images
```

Comment:
why do you introduce the low frequency into the masked image and not in the original image?
Might be cleaner if we create the original image, and the original image + low frequency, the from there we create the masked versions

Reply:
This is also done. With the new names, it's now easier to see